values/common.yaml

```diff
@@ -0,0 +1,30 @@
+securityContext:
```

Comment:
Should we codify a naming pattern for values files now that its being changed/deviated from here? In all of SWF and uds-core the values files are named similarly and I guess that has changed recently in some of the other repos (like artifactory). I understand *assuming* someone has the visual context of the `values` directory the `-values` suffix is kind of redundant (less so if their view doesnt have that context), but would greatly prefer that we align the way we name things across repos/packages.
![image](https://github.com/user-attachments/assets/48b52b25-c8b6-42e5-8fc6-a4dd4bd0f70d)
![image](https://github.com/user-attachments/assets/37e5b083-ed28-46a4-b52a-311f5e53f2eb)
![image](https://github.com/user-attachments/assets/6aee456c-6563-4305-8d9b-1d29e0d5cada)
![image](https://github.com/user-attachments/assets/edc92f68-644d-4185-9277-1df9072d0e9d)
![image](https://github.com/user-attachments/assets/e88a2ab0-006d-4aa9-a0af-2eef8c6a5a35)

Reply:
added `-values`